### ISSUE TITLE
Flipping output image

### ISSUE THREAD

**Issue by tomlankhorst** [ethz-asl/libvisensor#8]
Is this VI-sensor interfacing library capable of flipping the image upside-down or left-right and if so, how to configure it.

**Comment by schneith** [comment]
Yes, this is possible. I had a quick look and you might be able to do it with this call:
https://github.com/ethz-asl/libvisensor/blob/master/include/visensor/visensor_api.hpp#L67
The ```cmd``` argument for this would be ```row_flip``` and ```column_flip```.

In general, you can set any register of the MT9V034 camera chip that has been configured here:
https://github.com/ethz-asl/libvisensor/blob/master/include/sensors/camera_mt9v034.hpp#L63
